x/leverage/keeper/oracle.go

```diff
@@ -11,6 +11,11 @@ import (
 
 var ten = sdk.MustNewDecFromStr("10")
 
+// TODO: Parameterize and move this
+const (
+	minHistoracleStamps = uint64(24)
```

Comment:
Why no adding this directly to `Params`? 
Could you add a task to linear for this, and include link here?

Reply:
Follow-up task added [to linear](https://linear.app/umee/issue/UMEE-253/leverage-historacle-params-and-migration) (includes migration)